src/api/app/helpers/webui/request_helper.rb

```diff
@@ -139,4 +142,47 @@ def diff_data(action_type, sourcediff)
   def diff_label(diff)
     "#{diff['project']} / #{diff['package']} (rev #{diff['rev']})"
   end
+
+  # rubocop:disable Style/FormatStringToken, Style/FormatString
```

Comment:
One of them prefers `%<foo>`  over `%{foo}` but that causes argument errors. The other is configured to use `format` instead of `String#%`.
Since applying this would make the whole statement messier, I left it as is.

Reply:
What causes argument errors? :confused:

Reply:
[nitpick] What about regenerating the todo instead of excluding it here with the idea of solving this at some point in the future?